MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -799,23 +799,25 @@ private StreamInfo BuildVideoItem(MediaSourceInfo item, MediaOptions options)
             string? audioCodec)
         {
             // Prefer matching video codecs
-            var videoCodecs = ContainerProfile.SplitValue(videoCodec);
+            List<string> videoCodecs = new();
+            if (videoStream is not null && videoCodec.ContainsContainer(videoStream.Codec))
+            {
+                videoCodecs.Add(videoStream.Codec!);
+            }
 
-            // Enforce HLS video codec restrictions
-            if (string.Equals(playlistItem.SubProtocol, "hls", StringComparison.OrdinalIgnoreCase))
+            if (videoCodec is not null)
```

Comment:
If it's null, wouldn't it crash on line 803?

Reply:
We are checking for non-null there too.